mnefun/_mnefun.py

```diff
@@ -1706,6 +1707,21 @@ def _cals(raw):
         return raw.cals
 
 
+def _get_fir_kwargs(fir_design):
+    """Get FIR kwargs in backward-compatible way."""
+    fir_kwargs = dict()
+    if fir_design != 'firwin2':
+        # only pass if necessary (backward compat)
+        fir_kwargs.update(fir_design=fir_design)
+    old_kwargs = dict()
+    if 'fir_design' in mne.fixes._get_args(mne.filter.filter_data):
+        old_kwargs.update(fir_design='firwin2')
+    elif len(fir_kwargs):
```

Comment:
You'll have to think about the various cases, i.e. the values of `fir_design` and the MNE version, and think about the flow. But in any case, I think it should work okay